examples/howto/Model_Builder.myst.md

```diff
@@ -0,0 +1,255 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Using ModelBuilder class for deploying PyMC models 
+:::{post} Sep 12, 2022
+:tags: model builder class, model class, linear model
+:category: Beginner
+:author: Shashank Kirtania, Thomas Wiecki
+:::
+
++++
+
+##### Motivation
```

Comment:
This should either be a level 2 heading (two # only) and then there should be more similar level 2 headings, otherwise it looks like everything below is inside this section which isn't really the case.

Or alternatively simply some bold text (not recommended though, having multiple section and subsection headings helps reading and navigation, especially when coming back to the doc or for sharing specific inisghts from it).